codegen/smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/rendering/ServiceClientGenerator.kt

```diff
@@ -41,6 +41,11 @@ class ServiceClientGenerator(private val ctx: RenderingContext<ServiceShape>) {
              * Context key for the service symbol
              */
             val ServiceSymbol: SectionKey<Symbol> = SectionKey("ServiceSymbol")
+
+            /**
+             * Context key for the SDK ID
+             */
+            val SdkId: SectionKey<String> = SectionKey("SdkId")
```

Comment:
**nit**: Maybe we should just put the entire context/settings into the writer context for future use cases?